mmv1/products/compute/StoragePoolType.yaml

```diff
@@ -0,0 +1,155 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'StoragePoolType'
+kind: 'compute#storagePoolType'
+description: |
+  Represents a StoragePoolType data source.
+  The type of Hyperdisk Storage Pool that you create determines the type of disks that you can create in the storage pool.
+references:
+  guides:
+    'Types of Hyperdisk Storage Pools': 'https://cloud.google.com/compute/docs/disks/storage-pools#sp-types'
+  api: 'https://cloud.google.com/compute/docs/reference/rest/v1/storagePoolTypes'
+base_url: 'projects/{{project}}/zones/{{zone}}/storagePoolTypes'
+self_link: 'projects/{{project}}/zones/{{zone}}/storagePoolTypes/{{storagePoolType}}'
+immutable: true
+exclude: true
```

Comment:
this file shouldn't be necessary since we are excluding it from generation.

Reply:
I added `ResourceRef` to `StoragePoolType` so I leave this file as is.